src/xcode/ENA/ENA/Source/Models/Exposure/__tests__/WarnOthersTests.swift

```diff
@@ -0,0 +1,56 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+@testable import ENA
+import ExposureNotification
+import XCTest
+
+class WarnOthersTests: XCTestCase {
+	
+	private var store: SecureStore!
+	
+	override func setUpWithError() throws {
+		store = try SecureStore(at: URL(staticString: ":memory:"), key: "123456", serverEnvironment: ServerEnvironment())
+	}
+	
+	func testWarnOthers_allVariablesAreInitial() throws {
```

Comment:
Could you split this up into smaller tests and add some more scenarios?

Reply:
Let me think on...

Reply:
Refactored 💥